pkgs/development/tools/ammonite/default.nix

```diff
@@ -1,16 +1,5 @@
-{ lib
-, stdenv
-, fetchurl
-, jre
-, writeScript
-, common-updater-scripts
-, git
-, nixfmt
-, nix
-, coreutils
-, gnused
-, disableRemoteLogging ? true
-}:
+{ lib, stdenv, fetchurl, jre, writeScript, common-updater-scripts, git, nixfmt
+, nix, coreutils, gnused, disableRemoteLogging ? true }:
```

Comment:
Do the bot runs some kinda of formatting? I didn't really like this change.

Reply:
Seems that the update script runs nixfmt:

https://github.com/NixOS/nixpkgs/blob/251c375aa0b412311d11dbf4d9dfe2c731f7d954/pkgs/development/tools/ammonite/default.nix#L64

Reply:
Well, this seems kinda unfortunate :(.